### ISSUE TITLE
Copy Code has no Effect 

### ISSUE THREAD

**Issue by SikandarJODD** [techniq/layerchart#65]
## No Effect on Copy Code
![image](https://github.com/techniq/layerchart/assets/93428946/a1149e1d-2545-4053-bc21-3b027c45167b)

After clicking on Copy code it should show -> **Copied!** 

This enhance the Developer Experience

**Comment by techniq** [comment]
Good catch.  I forgot to update to Svelte UX's [CopyButton](https://svelte-ux.techniq.dev/docs/components/CopyButton) for this case.  Updated and deployed.

https://github.com/techniq/layerchart/assets/177476/9b3ab4d2-dfa8-45d3-a1e3-14030310998f
